exchanges/ftx/ftx_wrapper.go

```diff
@@ -0,0 +1,873 @@
+}
+
+// GetWebsocket returns a pointer to the exchange websocket
+func (f *FTX) GetWebsocket() (*wshandler.Websocket, error) {
+	return f.Websocket, nil
+}
+
+// GetActiveOrders retrieves any orders that are active/open
+func (f *FTX) GetActiveOrders(getOrdersRequest *order.GetOrdersRequest) ([]order.Detail, error) {
+	var resp []order.Detail
+	for x := range getOrdersRequest.Pairs {
+		assetType, err := f.GetPairAssetType(getOrdersRequest.Pairs[x])
+		if err != nil {
+			return resp, err
+		}
+		var tempResp order.Detail
+		orderData, err := f.GetOpenOrders(f.FormatExchangeCurrency(getOrdersRequest.Pairs[x], assetType).String())
+		if err != nil {
+			return resp, err
+		}
+		for y := range orderData {
+			tempResp.ID = strconv.FormatInt(orderData[y].ID, 10)
+			tempResp.Amount = orderData[y].Size
+			tempResp.AssetType = assetType
+			tempResp.ClientOrderID = orderData[y].ClientID
+			tempResp.Date = orderData[y].CreatedAt
+			tempResp.Exchange = f.Name
+			tempResp.ExecutedAmount = orderData[y].Size - orderData[y].RemainingSize
+			tempResp.Pair = currency.NewPairFromString(orderData[y].Market)
+			tempResp.Price = orderData[y].Price
+			tempResp.RemainingAmount = orderData[y].RemainingSize
+			var orderVars OrderVars
+			orderVars, err = f.compatibleOrderVars(orderData[y].Side,
+				orderData[y].Status,
+				orderData[y].OrderType,
+				orderData[y].FilledSize,
+				orderData[y].Size,
+				orderData[y].AvgFillPrice)
+			if err != nil {
+				return resp, err
+			}
+			tempResp.Status = orderVars.Status
+			tempResp.Side = orderVars.Side
+			tempResp.Type = orderVars.OrderType
+			tempResp.Fee = orderVars.Fee
+			resp = append(resp, tempResp)
+		}
+		triggerOrderData, err := f.GetOpenTriggerOrders(f.FormatExchangeCurrency(getOrdersRequest.Pairs[x], assetType).String(), getOrdersRequest.Type.String())
+		if err != nil {
+			return resp, err
+		}
+		for z := range triggerOrderData {
+			tempResp.ID = strconv.FormatInt(triggerOrderData[z].ID, 10)
+			tempResp.Amount = triggerOrderData[z].Size
+			tempResp.AssetType = assetType
+			tempResp.Date = triggerOrderData[z].CreatedAt
+			tempResp.Exchange = f.Name
+			tempResp.ExecutedAmount = triggerOrderData[z].FilledSize
+			tempResp.Pair = currency.NewPairFromString(triggerOrderData[z].Market)
+			tempResp.Price = triggerOrderData[z].AvgFillPrice
+			tempResp.RemainingAmount = triggerOrderData[z].Size - triggerOrderData[z].FilledSize
+			tempResp.TriggerPrice = triggerOrderData[z].TriggerPrice
+			orderVars, err := f.compatibleOrderVars(triggerOrderData[z].Side,
+				triggerOrderData[z].Status,
+				triggerOrderData[z].OrderType,
+				triggerOrderData[z].FilledSize,
+				triggerOrderData[z].Size,
+				triggerOrderData[z].AvgFillPrice)
+			if err != nil {
+				return resp, err
+			}
+			tempResp.Status = orderVars.Status
+			tempResp.Side = orderVars.Side
+			tempResp.Type = orderVars.OrderType
+			tempResp.Fee = orderVars.Fee
+			resp = append(resp, tempResp)
```

Comment:
Please redeclare tempResp at the start of the loops in here, otherwise you're modifying the same struct and appending it to the list. Could cause potential issues like above (but I have no active orders to verify)

Reply:
This isn't required after I tested the `append` function to verify.